api/src/main/resources/client_api.yaml

```diff
@@ -74,6 +74,57 @@ paths:
           schema:
             $ref: "#/definitions/MaterializeCohortResponse"
 
+  /v1/workspaces/{workspaceNamespace}/{workspaceId}/concepts:
+      get:
+        security: []
+        tags:
+          - concepts
+
+        description: >
+          Searches for concepts in concept table by name, and optionally filter on
+          domain ID, vocabulary ID, or standard concept status. Uses the CDR version affiliated
+          with the workspace specified in the path.
+
+        operationId: "searchConcepts"
+        parameters:
+          - in: query
+            name: query
+            type: string
+            required: true
+            description: >
+              A query string that can be used to match a subset of the name (case-insensitively),
+              the entire code value (case-insensitively), or the concept ID.
+          - in: query
+            name: standard_concept
+            type: boolean
+            required: false
+            description: >
+              True if only standard concepts should be returned, false if any concepts
+              can be returned.
+          - in: query
+            name: vocabulary_id
+            type: string
+            required: false
+            description: The vocabulary ID for the concepts returned (e.g. SNOMED, RxNorm)
+          - in: query
+            name: domain_id
+            type: string
+            required: false
+            description: >
+             The domain ID for the concepts returned (e.g. Observation, Drug). TODO: consider
+             changing this to be an enum which is mapped to underlying domain ID value(s).
+          - in: query
```

Comment:
note: pagination is difficult to add later, as it is a breaking change. Even if the initial implementation is simplistic. Consider adding it

Reply:
OK, I'll think about this. I was of the mind that there isn't a good use case for returning more than one (largish) page of results here, and doing whatever paging you want to do client-side. And the current use case (displaying the results in a data frame in a notebook) definitely doesn't lend itself to pagination. But I could imagine adding pagination here anyway, just in case.